packages/phone-number-privacy/common/package.json

```diff
@@ -16,7 +16,7 @@
     "lib/**/*"
   ],
   "dependencies": {
-    "@celo/contractkit": "0.4.15-dev",
+    "@celo/contractkit": "0.4.16-dev",
     "@celo/utils": "0.1.21-dev",
```

Comment:
I think that this is the one making the `dependencies install` job fail

Reply:
looks like it also needs to be bump to `0.1.22-dev`

Reply:
oops my bad!